### DISCUSSION TITLE
Add notes to a tasks

### DISCUSSION THREAD

**Discussion by redhat-raptor** [ransome1/sleek#345] [Ideas]
As a user I would like to add notes to tasks so that I could store important information in the task, specially applicable to recurring tasks. Example:
```
(A) My long running todo item rec:w
<date>
something worth attaching to the task
some more note
 
<date>
something happened last week
some more note
```

**Comment by shbach** [comment]
https://github.com/ransome1/sleek/issues/338

**Comment by Saturo78** [comment]
I also would like to save some notes, especially the progress and next steps to a task.
But how could this be done?
Maybe there could be a link to a separate text file?

**Comment by aisbergde** [comment]
Some years ago I also wanted to add notes to items. But the main issue is: This is not supported by different apps, or it is supported only in an app specific way. And then the todo.txt will lost universality. And one major reason to use the todo.txt is the independence from specific software.

For me the main question was and is (and I was not able to answer): How to imagine a way, to add notes, which could work, using different apps on different devices?

BTW, the main reason, why I have to use Google tasks in parallel to todo.txt - only Google task allow me to add Emails as task and later to jump from the task to the email. And this works on the desktop in the web and on Android. And I gave up to try, to combine this with other tasks. Of course, a link from a todo.txt-task to a Google Task would be fine, but how?

In General I think, linking Notes or files or whatever should use some URI or URL, which works in General on the desktop and on mobile devices. Linking to Emails inside Google tasks is just an example, how this should work.

**Reply by Jamlee** [reply]
agree. i only focus what/where/when/time-spent
